### PR TITLE
Adding blog

### PR DESCRIPTION
**Pull request by KM-Kinter**

### CODE REVIEW

**Comment by reinhrst** [comment]
Looks good to me! @DoomHammer ready to merge?

**Review by DoomHammer — APPROVED** [review]
Who am I to disagree?

**Comment by reinhrst** [comment]
Merged

File: content/pl/blog/przeprowadzka.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Z ~45m^2 do ~100m^2 w niecały rok! Przestrzeń wykorzystana absolutnie w każdym calu na rozwój i pracę nad naszymi projektami.
+
+Chcemy Wam za to podziękować. To dla Was i dzięki Wam to wszystko robimy. To NASZA WSPÓLNA, przyjazna, pomocna i otwarta przestrzeń. Świetnie jest być częścią takiej kreatywnej hackerskiej społeczności!
+
+Co ta przeprowadzka oznacza dla Was?
+
+Więcej miejsca na projekty. Więcej miejsca na nowe narzędzia z których wspólnie możemy korzystać. Większe możliwości na organizację spotkań, warsztatów, hackathonów, itp. No i może więcej stołów na pizzę...
+
+Nowe możliwości to także nowe marzenia i nowe zachcianki. Jeśli chcielibyście dorzucić coś do naszej kolektywnej pracowni to zerknijcie co znajduje się na {{% link ref=/zasoby/chciejlista %}}Chciejliście{{% /link %}}.
```
[the remark this answers is not in the document]
Comment (DoomHammer) [reply content/pl/blog/przeprowadzka.md:30]: I realized the answer to this exatly 99 seconds after I posted the question... Rubber duck with exponential backoff or something...

**Pull request merged**